checker/tests/nullness-extra/Makefile

```diff
@@ -1,7 +1,7 @@
-.PHONY: all Bug109 multiple-errors package-anno issue265 issue309 compat shorthand issue502 issue559 issue594
+.PHONY: all Bug109 multiple-errors package-anno issue265 issue309 compat shorthand issue502 issue559 issue594 issue607
 
 # This skips tests that are not currently passing
-all: Bug109 compat issue265 issue309 issue502 multiple-errors package-anno shorthand 
+all: Bug109 compat issue265 issue309 issue502 multiple-errors package-anno shorthand issue607
```

Comment:
Because the test is currently failing: add a new .PHONY target "skipped" that only contains issue607.
Then we can still run the test target, but the failure won't get executed with "all".